util/stmtsummary/evicted.go

```diff
@@ -190,3 +216,221 @@ func (seElement *stmtSummaryByDigestEvictedElement) toEvictedCountDatum() []type
 func (ssMap *stmtSummaryByDigestMap) ToEvictedCountDatum() [][]types.Datum {
 	return ssMap.other.ToEvictedCountDatum()
 }
+
+func (ssbde *stmtSummaryByDigestEvicted) toCurrentDatum() []types.Datum {
+	var seElement *stmtSummaryByDigestEvictedElement
+	ssbde.Lock()
+	defer ssbde.Unlock()
+
+	if ssbde.history.Len() > 0 {
+		seElement = ssbde.history.Back().Value.(*stmtSummaryByDigestEvictedElement)
+	}
+
+	if seElement == nil {
+		return nil
+	}
+	induceSsbd := &stmtSummaryByDigest{
+		schemaName:    "other",
```

Comment:
Better to use a constant variable to make it easier to modify.

How about rename to `other_summary`?

Reply:
@breeswish PTAL

Reply:
we can follow MySQL's way I think.

Reply:
Ok, let's follow MySQL's behaviour